quality/metrics/bleu.md

```diff
@@ -0,0 +1,31 @@
+---
+grand_parent: Building and research
+parent: Metrics
+nav_order: 1
+title: BLEU
+description: BiLingual Evaluation Understudy
+---
+
+**BLEU** (BiLingual Evaluation Understudy) is the standard metric for machine translation evaluation.
+
+> The closer a machine translation is to a professional human translation, the better it is.
+
+BLEU compares machine translation output to a single reference translation.
+BLEU is an [n-gram](/../concepts/n-gram.md) word-based metric.
+
+BLEU is based on translation precision.
+Its basic unit of evaluation is the sentence.
+
+BLEU treats all n-grams equally.
+
+### Flavours
+
+- sacreBLEU
```

Comment:
I would say that sacreBLEU is not a metric, it is an implementation of a few metrics: BLEU, chrF and TER.